src/pages/frontpage/FrontPageHeading.tsx

```diff
@@ -0,0 +1,17 @@
+import { Box, Typography, Container } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+
+export const FrontPageHeading = () => {
+  const { t } = useTranslation();
+
+  return (
+    <Container maxWidth="md" sx={{ display: 'flex', justifyContent: 'center', pt: '3rem' }}>
+      <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1.5rem' }}>
+        <Typography variant="h1" sx={{ fontSize: '3rem', color: '#120732' }}>
+          {t('common.page_title')}
+        </Typography>
+        <Typography sx={{ fontSize: '1rem' }}>{t('search_in_national_research_publication')}</Typography>
+      </Box>
+    </Container>
```

Comment:
`Container` ser overflødig ut her?
```suggestion
      <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1.5rem' }}>
        <Typography variant="h1" sx={{ fontSize: '3rem', color: '#120732' }}>
          {t('common.page_title')}
        </Typography>
        <Typography sx={{ fontSize: '1rem' }}>{t('search_in_national_research_publication')}</Typography>
      </Box>
```